.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+version: 2
+jobs:
+  build:
+    machine:
+      image: circleci/classic:latest
+
+    # Get rid of existing virtualenvs on circle ci as they conflict with conda.
+    # Trick found here:
+    # https://discuss.circleci.com/t/disable-autodetection-of-project-or-application-of-python-venv/235/10
+    steps:
+      - checkout
+      - run: cd && rm -rf ~/.pyenv && rm -rf ~/virtualenvs
+      # We need to remove conflicting texlive packages.
+      - run: sudo -E apt-get -yq remove texlive-binaries --purge
+      # Installing required packages for `make -C doc check command` to work.
+      - run: sudo -E apt-get -yq update
+      - run: sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install dvipng texlive-latex-base texlive-latex-extra wget
+      - save_cache:
+          key: v1-datasets-{{ .Branch }}
+          paths:
+            - ~/nilearn_data
+
+      # Moving to nilearn directory before performing the installation.
+      - run: cd nilearn
+#      - run: wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O ~/miniconda.sh
+#      - run: chmod +x ~/miniconda.sh && ~/miniconda.sh -b
+#      - run: export PATH=$HOME/miniconda2/bin:$PATH
+#      - run: echo $PATH
+#      - run: conda update --quiet --yes conda
+      - run:
+          command: ./continuous_integration/install.sh
+          environment:
```

Comment:
Try to setup this environment right after machine or docker image. It should not give ```wget command not found``` error.

Reply:
I tried to build this locally. Atleast I am not getting the weird wget command not found error after building an environment before.